homeassistant/util/unit_conversion.py

```diff
@@ -266,6 +266,7 @@ class EnergyConverter(BaseUnitConverter):
         UnitOfEnergy.KILO_JOULE: _WH_TO_J,
         UnitOfEnergy.MEGA_JOULE: _WH_TO_J / 1e3,
         UnitOfEnergy.GIGA_JOULE: _WH_TO_J / 1e6,
+        UnitOfEnergy.MILLIWATT_HOUR: 1e6,
```

Comment:
```suggestion
        UnitOfEnergy.MILLI_WATT_HOUR: 1e6,
```
to keep it consistent with the other energy units (which are not consistent with other units :upside_down_face: which have no underscore)